core/azure-core-management/src/main/java/com/azure/core/management/implementation/AzureProxy.java

```diff
@@ -54,12 +57,13 @@
     /**
      * Create a new instance of RestProxy.
      * @param httpPipeline The HttpPipeline that will be used by this AzureProxy to send HttpRequests.
-     * @param serializer The serializer that will be used to convert response bodies to POJOs.
+     //* @param serializer The serializer that will be used to convert response bodies to POJOs.
```

Comment:
@hemanttanwar Rather than comment out lines like this, just delete them so that they don't linger.